tests/test_auth.py

```diff
@@ -287,6 +326,13 @@ def user_mgt_app():
     yield app
     firebase_admin.delete_app(app)
 
+@pytest.fixture(scope='module')
+def user_mgt_app_for_verify_token():
```

Comment:
Can we just modify the existing `user_mgt_app` fixture to have the `project_id` option?